src/components/City.js

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from "react";
+import {HashRouter, NavLink, Route} from "react-router-dom";
+
+
+class City extends Component {
+    render() {
+
+        // JAK ZROBIĆ WARUNKOWE RENEDEROWANIE W TYM MIEJSCU ??
```

Comment:
looks like you want to pick-up `name` URL param. Remember that it is on the same `params` object as `city`, it is not "sub-object"

Take it like this:
```js
const { name } = this.props.match.params;
// OR
const name = this.props.match.params.name;
```